src/h5p-vocabulary-drill.ts

```diff
@@ -56,6 +56,10 @@ class VocabularyDrill
     containerElement.appendChild(wrapper);
     containerElement.classList.add("h5p-vocabulary-drill");
 
+    if (!showTips) {
+      containerElement.classList.add("h5p-vocabulary-drill-hide-tips");
+    }
+
```

Comment:
We might want to remove the hints from the output, in case the underlying content types do something special when there are no hints.